### ISSUE TITLE
v0.2.8 of python-libcombine breaks ability to set and get modified dates

### ISSUE THREAD

**Issue by jonrkarr** [sbmlteam/libCombine#33]
v0.2.8 of the python version of libCOMBINE breaks the ability to set and get modified dates.

Here's an example which works with 0.2.7, but not with 0.2.8. Example file: [test.omex.zip](https://github.com/sbmlteam/libCombine/files/5691506/test.omex.zip)
```python
import libcombine
combine_archive_filename = 'test.omex'
combine_archive = libcombine.CombineArchive()
combine_archive.initializeFromArchive(combine_archive_filename)
location = combine_archive.getAllLocations()[0]
metadata = combine_archive.getMetadataForLocation(location)
for modified in metadata.getModified(): 
    print(modified.getDateAsString())
```

Appending modified dates has also been broken.
```python
date = libcombine.Date()
date.setDateAsString('...')
metadata.getModified().append(date)
```

**Comment by fbergmann** [comment]
Thanks, this is fixed with commit 73e20d760f1061126090cbc41bfba1a2789c788f, i'll re-release

**Comment by jonrkarr** [comment]
Thanks!
